packages/synapse-react-client/src/components/CardDeck/CardDeck.Mobile.tsx

```diff
@@ -4,45 +4,57 @@ import { ImageFileHandle } from '../widgets/ImageFileHandle'
 import { CardDeckDataProps } from './CardDeck'
 
 export default function CardDeckMobile({
-  //link,
   description,
   title,
   titleIconFileHandleAssociation,
   ctaButtonText,
   ctaButtonURL,
+  cardDeckType,
 }: CardDeckDataProps) {
   const titleElement = (
-    <div className="Programs__Mobile__Header">
-      <span className="Programs__Card__header__icon">
+    <div className="CardDeck__Mobile__Header">
+      <span className="CardDeck__Card__header__icon">
         {titleIconFileHandleAssociation && (
           <ImageFileHandle
             fileHandleAssociation={titleIconFileHandleAssociation}
           />
         )}
       </span>
-      <span className="Programs__Mobile__Header__Title"> {title} </span>
+      <span className="CardDeck__Mobile__Header__Title"> {title} </span>
     </div>
   )
   const content = (
-    <div className="Programs__Mobile__Content">
-      {/* {link && (
-        <p className="Programs__Card__header__info__link">
-          <a
-            className="highlight-link"
-            href={link}
-            target="_blank"
-            rel="noreferrer"
-          >
-            Visit website
-          </a>
-        </p>
-      )} */}
+    <div className="CardDeck__Mobile__Content">
       <p>{description}</p>
       <LargeButton
         color="secondary"
-        variant="contained"
+        variant={cardDeckType === 'cckp' ? 'outlined' : 'contained'} // Update variant based on cardDeckType
         href={ctaButtonURL}
-        sx={{ marginLeft: 0 }}
+        sx={{
+          marginLeft: 0,
+          ...(cardDeckType === 'b2ai' && {
+            width: '200px',
+            fontSize: '14px',
+            backgroundColor: 'white',
+            boxShadow: 'none',
+            color: 'primary.main',
+            '&:hover': {
+              backgroundColor: 'primary.main',
+              color: 'white',
+            },
+          }),
+          ...(cardDeckType === 'cckp' && {
+            width: '170px',
+            fontSize: '14px',
+            boxShadow: 'none',
+            borderColor: 'primary.main',
+            color: 'primary.main',
+            '&:hover': {
+              backgroundColor: 'primary.main',
+              color: 'white',
+            },
```

Comment:
is `cardDeckType` ever undefined or a different value? if one of these is always true, then I think we can just apply the common styles unconditionally
```suggestion
          fontSize: '14px',
          boxShadow: 'none',
          color: 'primary.main',
          '&:hover': {
              backgroundColor: 'primary.main',
              color: 'white',
          },
          ...(cardDeckType === 'b2ai' && {
            width: '200px',
            backgroundColor: 'white',
          }),
          ...(cardDeckType === 'cckp' && {
            width: '170px',
            borderColor: 'primary.main',
          })
```